src/components/application_manager/test/policy_handler_test.cc

```diff
@@ -3126,6 +3126,95 @@ TEST_F(PolicyHandlerTest, GetEnabledLocalApps_SUCCESS) {
   EXPECT_EQ(enabled_local_apps, policy_handler_.GetEnabledLocalApps());
 }
 
+TEST_F(PolicyHandlerTest, PushAppIdToPTUQueue_PolicyEnabled_SUCCESS) {
+  ChangePolicyManagerToMock();
+  const uint32_t expected_apps_count = 1u;
+  EXPECT_CALL(*mock_policy_manager_, UpdatePTUReadyAppsCount(_));
+  policy_handler_.PushAppIdToPTUQueue(kAppId1_);
+  EXPECT_EQ(expected_apps_count,
+            policy_handler_.applications_ptu_queue_.size());
+}
+
+TEST_F(PolicyHandlerTest, PushAppIdToPTUQueue_PolicyDisabled_FAIL) {
+  ChangePolicyManagerToMock();
```

Comment:
@YarikMamykin AICS you are calling `ChangePolicyManagerToMock();` in every test case. Is it worth to move it to SetUp() instead of calling in every test case?

Reply:
@AByzhynar actually not all tests use `ChangePolicyManagerToMock();`. More than that, I gave a shot and tried to move it inside SetUp method and this caused segmentation fault in some tests. So unfortunately it has to stay this way.